docs/README.md

```diff
@@ -28,6 +28,7 @@ Here follows the list of GitHub Actions topics available in the current document
     - [Retry failing step](#retry-failing-step)
     - [SSH debug](#ssh-debug)
     - [Triggering a workflow in another repository](#triggering-a-workflow-in-another-repository)
+    - [Generate Dependabot Glob Action](#generate-dependabot-glob-action)
```

Comment:
Maybe it's not obvious but there are ordered alphabetically: do you mind moving it please?